app/assets/src/components/views/SampleUploadFlow/SampleUploadFlowHeader.jsx

```diff
@@ -47,7 +47,7 @@ class SampleUploadFlowHeader extends React.Component {
                 <div className={cs.subtitle}>
                   Rather use our command-line interface?
                   <a
-                    href="/cli_user_instructions"
+                    href="https://help.idseq.net/hc/en-us/articles/360034790414-Upload-with-Command-Line"
```

Comment:
this should also change to use `<ExternalLink>` ;) I think you saw ESLint flag it `51:21  error  Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank`